src/OrbitGl/TimeGraph.cpp

```diff
@@ -375,6 +379,70 @@ void TimeGraph::ProcessValueTrackingTimer(const TimerInfo& timer_info) {
   }
 }
 
+void TimeGraph::ProcessMemoryTrackingTimer(const TimerInfo& timer_info) {
+  const std::string kUnusedLabel = "System Memory Unused (MB)";
+  const std::string kBuffersOrCachedLabel = "System Memory Buffers / Cached (MB)";
+  const std::string kUsedLabel = "System Memory Used (MB)";
+  const std::string kWarningThresholdLabel = "Production Limit";
+  const std::string kValueUpperBoundLabel = "System Memory Total";
+  const std::string kValueLowerBoundLabel = "Minimum: 0 GB";
+  const std::string kTrackValueLabelUnit = "MB";
+
+  constexpr double kValueLowerBoundRawValue = 0.0;
+  constexpr uint8_t kTrackValueDecimalDigits = 2;
+  constexpr uint64_t kKilobytesToBytes = 1024;
+  constexpr double kMegabytesToKilobytes = 1024.0;
+
+  CHECK(app_->GetCollectMemoryInfo());
+  uint64_t warning_threshold_kb = app_->GetMemoryWarningThresholdKb();
+  std::string warning_threshold_pretty_size =
+      GetPrettySize(warning_threshold_kb * kKilobytesToBytes);
+  std::string warning_threshold_pretty_label =
+      absl::StrFormat("%s: %s", kWarningThresholdLabel, warning_threshold_pretty_size);
+  double warning_threshold_raw_value =
+      static_cast<double>(warning_threshold_kb) / kMegabytesToKilobytes;
+
+  int64_t total_kb = orbit_api::Decode<int64_t>(timer_info.registers(OrbitApp::kTotalKb));
+  std::string total_pretty_label = "";
```

Comment:
nit
```suggestion
  std::string total_pretty_label;
```
You can rely on the default constructor here

Reply:
Done!